src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +55,75 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("모든 지하철역 목록을 조회한다.")
+    @Test
+    void readStations() {
+        // Given
+        createStations();
+        int expectedCountOfStation = 2;
+
+        // When
+        List<String> stationsNames = RestAssured.given().log().all()
+                .when().get("/stations")
+                .then().log().all()
+                .extract().jsonPath().getList("name", String.class);
+
+        // Then
+        assertThat(stationsNames).contains("숙대입구", "서울역");
+        assertThat(stationsNames.size()).isEqualTo(expectedCountOfStation);
```

Comment:
```suggestion
        assertThat(stationsNames).containsExactly("숙대입구", "서울역");
```

containsExactly()를 사용하면, expectedCountOfStation는 검증하지 않아도 좋아보여요!

containsExactly와 containsExactlyInAnyOrder에 대해서 학습해 볼까요?